backend/tests/features/carrinho_compras.feature

```diff
@@ -0,0 +1,31 @@
+Feature: carrinho de compras
+    As eu sou um cliente
+    I want escolher os itens que desejo comprar
+    So that eu possa efetuar e receber meu pedido
+
+Scenario: inserir um item no carrinho de compras
+    Given O array "carrinho_compras" do "users/1"
+    When "GET /menu/0" é solicitado
```

Comment:
The scenario try add an item in user cart, then, the correct method should be "POST" instead "GET" method.

/menu/0 should return an item of menu, replace for
`When "usuario1" faz uma requisição "GET" para rota "/order/0" `